### ISSUE TITLE
App is not accessible for screen readers 

### ISSUE THREAD

**Issue by psanford** [psanford/wormhole-william-mobile#8]
Currently the application is inaccessible to screen readers like TalkBack.

I believe this is a current limitation with Gio. I'll check with them on where accessibility features fall on their roadmap.

**Comment by psanford** [comment]
From the #gioui channel in the gophers Slack, support for screen readers is one of the requirements for the Gio 1.0 stable release. We'll track those changes as they come. I'll also look to see if I can assist with any of that upstream work.

**Comment by psanford** [comment]
Initial accessibility support has landed in Gio: https://lists.sr.ht/~eliasnaur/gio/%3CCG458RGLBR08.2DD5JAQH1MQRJ%40macbog.local%3E

**Comment by psanford** [comment]
This is in release v1.0.7
